.yamato/upm-ci.yml

```diff
@@ -130,16 +96,39 @@ validate_{{ platform.name }}_{{ editor.version }}:
 {% endfor %}
 {% endfor %}
 
+# The following job runs the tests to validate the the public API files are up to date.
```

Comment:
duplicate `the` on this line after validate

Reply:
Good catch. I will change it.